src/main/webapp/app/admin/standardized-competencies/admin-standardized-competency.service.ts

```
@@ -8,7 +8,9 @@ import { KnowledgeAreaDTO, KnowledgeAreasForImportDTO, StandardizedCompetencyDTO
 export class AdminStandardizedCompetencyService {
     private resourceURL = 'api/admin/standardized-competencies';
 
-    constructor(private httpClient: HttpClient) {}
+    private httpClient = inject(HttpClient);
+
+    constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)